entrypoint.sh

```diff
@@ -11,8 +11,8 @@ credentials="${awsDir}/credentials"
 
 mkdir -p "${awsDir}"
 
-echo -e "[profile default]\noutput = json" >> "$config"
-tokendito --aws-profile default -ou $INPUT_OKTA_APP_URL -R $INPUT_AWS_ROLE_ARN --username $INPUT_OKTA_USERNAME --password $INPUT_OKTA_PASSWORD --mfa-method ${INPUT_OKTA_MFA_METHOD:=token:software:totp} --mfa-response `echo $INPUT_OKTA_MFA_SEED | mintotp` >> /dev/null
+echo -e "[profile default]\noutput = json" >>"$config"
+tokendito --aws-profile default -ou $INPUT_OKTA_APP_URL -R $INPUT_AWS_ROLE_ARN --username $INPUT_OKTA_USERNAME --password $INPUT_OKTA_PASSWORD --mfa-method ${INPUT_OKTA_MFA_METHOD:=token:software:totp} --mfa-response $(echo $INPUT_OKTA_MFA_SEED | mintotp ${INPUT_TOTP_TIME_SLOT:=5}) >>/dev/null
```

Comment:
The `:=5` part is really necessary if in action.yml we are defaulting to 5?

Reply:
Yeah. Whatever you have in the `action.yml` is not passed. It's merely for documentation. Welcome to the quirks of Github Actions!

Reply:
See this open issue: https://github.community/t/actions-input-parameters-are-not-passed-to-images-pulled-from-a-docker-repository/16473